content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Virtual Machine Installation
+description: Deploy istio and connect a virtual machine to it.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to deploy Istio and connect a virtual machine to it.
+
+{{< warning >}}
+This guide relies heavily on the
```

Comment:
I think we can rephrase this a bit. This guide has a requirement that the user is using a plugin root CA and has configured Citadel as an intermediate CA.

Basically I'm trying to make sure we're clear to the reader what the requirements are on the Istio install:
1. Setup the control plane with plugin root CA and an intermediate cert
2. Turn on mesh expansion

I think it is fine to explain how to do those as part of the guide but we want to make it easier for the user to understand how things fit together.

Also the linked-to-task needs to be updated to use the Makefile instead of samples (hopefully already being addressed).

Reply:
Agree.

I have changed the install flow slightly and added the words "install or upgrade".

I changed the warning as follows:
```
This guide has a requirement that the user is using a [plugin root CA](/docs/tasks/security/cert-management/plugin-ca-cert/)
and has configured Istio as an intermediate CA.        
```

Only real change from your suggestion is instead of Citadel, using the word Istio. as component names ideally should not be used in documentation.